src/wfpak.cpp

```diff
@@ -1819,11 +1819,13 @@ RC WFILE::wf_Close( 		// Close weather file if open
 {
 	RC rc = RCOK;
 	if (yac)
-	{	if ( hdr				// if header given
+	{	
```

Comment:
I thought we agreed we wouldn't use SHAs in comments like this. This should be a TODO (MP) like everything else.